zerver/tests/tests.py

```diff
@@ -1548,6 +1548,24 @@ def check_for_toggle_param(self, pattern, param):
         user_profile = get_user_profile_by_email("hamlet@zulip.com")
         self.assertEqual(getattr(user_profile, param), False)
 
+    def check_for_toggle_param_patch(self, pattern, param):
+        # type: (str, str) -> None
+        self.login("hamlet@zulip.com")
+        user_profile = get_user_profile_by_email("hamlet@zulip.com")
```

Comment:
For a true test here, we should either assert the value starts False or force it to be False.  Otherwise, you can get false positives where you think your endpoint is doing the write job of setting the value, but the value was already true.